### ISSUE TITLE
9

### ISSUE THREAD

**Issue by szwnba** [szwnba/szwnba.github.io#15]
Creating Your PopAds Campaign

So far, we’ve setup our BeMob to be able to track PopAds, added MOBIPIUM as an affiliate network, and added 5 offers we want to include in our first campaign.

Now, we get to actually setup that campaign.

First, let’s create our Campaign URL within BeMob and then we’ll create our new campaign in PopAds.

Creating a BeMob Campaign
Creating a new campaign in BeMob is pretty simple and it’s even easier in this guide because we’ve already done most of the work by creating our offers, adding our affiliate network, and adding the traffic source.

Now, we just need to create a campaign with all of those things so we can get our Campaign URL to promote on PopAds.

Login to BeMob and click on the Campaigns link in the top navigation. Once there, click on New.

￼

The basic criteria for a campaign are pretty simple:
Name
Traffic source
Destination
There are other options as well, but those 3 things are the most important when creating any affiliate marketing campaign.

Campaign Name
You can use whatever naming conventions you want, but I’ve adopted a fairly simple way to name all of my campaigns that makes it very easy for me to find them when I am running reports and doing my analysis.

Country_Device_OS_Browser

I use the same name on the traffic source as I do on BeMob so I can easily find them and there’s no confusion.

So, for this campaign, I am going to use the name: WW_Android_Chrome

I left out device because I know Android is mobile, but if we end up split testing cellphone/tablets, I would add that to the name.

Traffic source
This should be obvious, but you select the traffic source you are going to run the campaign on. I may run this same campaign across multiple traffic sources at some point which is why you want to make sure you select the traffic source you are using for this particular campaign. It’s also important because BeMob will attach the important tracking variables/macros to our Campaign URL based on the traffic source we select.

Destination
Destination takes up the entire right panel of the campaign creation screen on BeMob because it is so important. This is where you choose the landing pages and offers we are going to drive traffic to. For this guide campaign, we will be direct linking so there will be no need to add or select a landing page. However, we’re going to split test offers so we need to make sure we choose all of them.

With all that being said, here is what my Create Campaign form looks like for this particular campaign:

￼

You’ll notice that I have Enabled checked for Sixth sense. I recommend you enable it simply so you can check it out once your campaign has some data. It’s not something specifically that I use all the time to optimize my campaigns, but since you are new to BeMob, I recommend you enable it just so you can check it out once you have some data.

The Cost model is set to Auto because the PopAds traffic source template that we created earlier is setup to track the cost of your campaign in BeMob using a custom tracking token. Not all traffic sources have this, but when they do, we want to use it as it’ll make it so we can easily analyze and optimize our campaigns based on the cost of every click without having to do any manual work.

For Destination, I’ve selected Built-in flow. This allows us to choose the offers we want to split test and then later make modifications to them really easily.

You should click on the LANDINGS toggle because we are not going to use any of our own landing pages for this campaign. We are going to direct link to the 5 offers we setup earlier.

Once you’ve toggled LANDINGS, click the Add offer green button to add 5 offers total. Then, click the dropdown and select each of the 5 offers.

Once each is selected and added to your flow, your form should look a lot like mine shown above.

Click the Save button to create/save your campaign. BeMob will show you a list of your Campaign Links. Click the green Copy button on the top one.

￼

Okay, now we are going to create your PopAds campaign for our new campaign we just created!

Login to your PopAds account and click on New Campaign under your Advertiser’s Panel.

￼

We are going to setup a campaign with this criteria:
$5 Budget (50% for initial testing in this case)
Worldwide (All Countries)
Android OS
Chrome 47+ Browser
Smartphone
We’re going to test worldwide Android traffic using the Chrome browser because that seems to be what converts best for push notification smartlinks. We’re targeting Chrome 47+ because that’s what supports push notifications.

The Create a New Campaign form on PopAds is pretty long, but I will break down each section here. Once you create a few campaigns you’ll be able to go through this lengthy form very quickly.

General Information
As I said, I use the same name on my campaign in BeMob as I do in my traffic source. The name of this campaign in BeMob for me is: WW_Android_Chrome

The URL is where we put the Campaign URL we have copied from BeMob. We’re going to send traffic to a single URL (we’re split testing within BeMob so PopAds just gets 1 link from us). Paste your Campaign URL into the URL section.

Everything else I would leave the same. A Frequency Cap of 1/24 is good (and what I pick on nearly all traffic sources), once the campaign is approved I want it to run, and everything else is fine.

￼

For now, skip Budgets and Throttling and go to Categories.

Categories
Adult traffic is typically cheaper and also does not always perform the best. I am not saying you should not test the adult traffic on PopAds, but for this example, let’s remove it. You can do this by clicking the Add All button and then typing Adult under Your choices, selecting all the categories shown in the box, and then clicking Remove.

￼

Now, if you delete where you typed Adult, you will see every other non-adult category showing as Your choices.

Country Targeting
As I said, we are going to target all countries with this campaign and then we’re remove the ones that are not profitable.

Simply click Add All and all the countries will be added to our targeting.

￼

We want to add all the countries to our choices so they are easier to remove later. If you did not click Add All, your campaign would still target all countries.

Skip Society Targeting for now.

Environmental Targeting
This is where we choose our Operating System and Browser. We want to target Android devices so type Android into the Available choices under Operating Systems and then click Add.

For browsers, we want to select Google Chrome 47+ so type Chrome into the Available choices under Browsers and then select Google Chrome / 47 and everything higher than that (Google Chrome / 48, 49, etc) and click Add.

￼

We’ll leave the Screen Resolutions as the default.

Device Targeting
The only thing we need to change with Device Targeting for now is selecting Smartphone under Form Factors.

We could also test Android Tablets (and maybe you will want to do that later), but for now, let’s just target smartphones to make our test easier to optimize.

Double click Smartphone from the Available choices to add it to Your choices. We will leave the Devices alone for now.

You can skip Connecting Targeting, Time Targeting, Website Targeting, Adscore Traffic Validation, and just click on Summary to open our Summary.

Summary
The Traffic Estimator in the Summary section of PopAds campaign creation is one of it’s coolest and most unique features. Many traffic sources have started to copy this feature, but I’ve always like how it’s presented with PopAds.

This summary gives you important information to help you with your bidding (which we skipped earlier).

￼

As you can see, the average bid for this traffic is 0.000562 (a $0.56 CPM). They show the highest bid, but all I really care about is the average.

A $0.56 CPM seems pretty cheap, but it’s important to know that the payouts on the offers we added are going to be pretty low. We’re hopefully going to make up for a low payout with a high conversion rate, but even then, this traffic will be hard to get profitable.

The point of this guide is not to run the most profitable campaign. It’s to get traffic and learn how to analyze and optimize it. So, that’s what we’re going to do.

The bid distribution map shows that there are some really low bids getting a nice amount of traffic:

￼  

So, let’s hit the top of that and set our bid at $0.00033 ($0.33 CPM).

Click on Budgets and let’s set our Max Bid and Daily budget.

Budgets
As I said, we’ll put our bid at 0.00033 and our Daily budget for this is going to be $5. We’ll make our Budget $5 as well and then increase this to $10 after we’ve collected some data. The first $5 is just for initial testing.

￼

Now, click on Summary again and the Summary will reload with the data base on our bid and budget:

￼

The map shows the countries that we will likely get most of our traffic from, but we’ll see what the data actually shows once our campaign is running.

Click Create Campaign and PopAds will make you confirm that you really want to run a worldwide campaign (we do). Check the box and click Create Campaign again.

￼

And…our campaign is pending approval!

But, we’re not done quite yet.

￼

Click on the Conversion Tracking button so we can setup our PopAds postback in BeMob.

NEXT: Step 9 - Setup PopAds Postback in BeMob
Attachments
